sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md

```diff
@@ -1,15 +1,24 @@
 # Release History
 
-## 4.12.0-beta.3 (Unreleased)
+## 4.12.0 (2022-07-08)
 
 ### Features Added
+- General availability of supporting publish CloudEvents to partner topics.
```

Comment:
Don't need to say "General availability" since we don't have beta in the version.
```suggestion
-  Added support to publish CloudEvents to partner topics by setting the channel name.
```